usefulUtilities/sitV2/appResources/appData/sitServerV2.js

```diff
@@ -157,13 +156,44 @@
     }
 
     // #endregion CAN SIT ZONE
+
+    /* Check children of the sit cube to make sure there is only one zone. If more are found, delete them */
+    function checkForExtraZones() {
+        var canSitZones = [];
+        Entities.getChildrenIDs(_this.entityID).forEach(function(childOfSitCube) {
+            var name = Entities.getEntityProperties(childOfSitCube, 'name').name;
+            if (name && name.indexOf("canSitZone") > -1) {
+                canSitZones.push(childOfSitCube);
+            }
+        });
+        if (canSitZones.length > 1) {
+            canSitZones.pop();
+            canSitZones.forEach(function(canSitZone) {
+                Entities.deleteEntity(canSitZone);
+            });
+        }
+    }
+
+    /* Set a random timeout with exponential growth to check for extra zones */
+    var ONE_SECOND_MS = 1000;
+    var ONE_TENTH_SECOND_MS = 100;
+    var EXPONENT = 2;
+    var timeUntilNextSound = Math.floor((Math.random() * ONE_SECOND_MS) + ONE_TENTH_SECOND_MS);
```

Comment:
I _think_ what you want here is a random value between 100ms and 1000ms, right? If so, the code below will do that. Is that what you wanted?
```suggestion
    var timeUntilNextSound = Math.floor(Math.random() * (ONE_SECOND_MS - ONE_TENTH_SECOND_MS)) + ONE_TENTH_SECOND_MS;
```

Reply:
I think it is fine if it gets a random number with a range of 1000ms beginning at100ms. Is there something wrong with the way it was done?

Reply:
no, there's nothing wrong with the way you did this. this seems like a fine implementation to me, i just didn't realize what the spec was :)

also, i just now noticed that this var name is `timeUntilNextSound` - please rename it to something relevant here